src/__tests__/pinning.test.js

```diff
@@ -231,6 +239,9 @@ describe('Pinning', () => {
       testClient.announceThread()
       await responsesPromise
       await dbSyncPromise
+      // for some reason there is an issue with the db not getting fully
+      // replicated in time even after the dbSyncPromise. Wait for 0.5 s
+      await new Promise((resolve, reject) => { setTimeout(resolve, 500) })
```

Comment:
hmm, only in tests, or think this would show up as an issues in production?

Reply:
This has to do with the code in the `testClient` afaik.